packages/utils/src/string.ts

```diff
@@ -15,13 +15,16 @@ export function stringStartsWith (string, startsWith) {
   return string.substring(0, startsWith.length) === startsWith
 }
 
-export function parseJson (jsonString) {
+export function parseJson<T = any> (jsonString: string): T | null {
   if (typeof jsonString === 'string') {
     jsonString = stringTrim(jsonString)
     if (jsonString) {
-      if (JSON && JSON.parse) // Use native parsing where available
-            { return JSON.parse(jsonString) }
-      return (new Function('return ' + jsonString))() // Fallback on less safe parsing for older browsers
+      if (JSON && JSON.parse) {
+        // Use native parsing where available
+        return JSON.parse(jsonString) as T;
+      }
+      // Fallback on less safe parsing for older browsers
+      return (new Function('return ' + jsonString))() as T
```

Comment:
Noting (aside) we should really rely on JSON as a polyfill, if it's not defined.

Reply:
yeah I think it's safe/wise to pull this out. It opens a vector to arbitrary code execution - if someone can unset the global JSON the fallback is dangerous.  We're relying on newer things than JSON, so this won't be the thing that breaks TKO.